openproblems/tasks/_batch_integration/batch_integration_graph/methods/combat.py

```diff
@@ -17,18 +17,18 @@ def combat_full_unscaled(adata):
     from scIB.preprocessing import reduce_data
 
     adata = runCombat(adata, "batch")
-    reduce_data(adata)
+    reduce_data(adata, umap=False)
     # Complete the result in-place
     return adata
 
 
 @method(
     method_name="Combat (hvg/unscaled)",
-    paper_name="Sc",
-    paper_url="temp",
-    paper_year=2020,
-    code_url="",
-    code_version=check_version("combatpy"),
+    paper_name="A test metric for assessing single-cell RNA-seq batch correction",
+    paper_url="https://www.nature.com/articles/s41592-018-0254-1",
+    paper_year=2018,
+    code_url="github.com/theislab/scanpy",
+    code_version=check_version("scanpy"),
```

Comment:
same issue with wrong reference... also maybe link the code url to the actual function in the scanpy repo?